ci-operator/step-registry/redhat-et/microshift/e2e/test/redhat-et-microshift-e2e-test-commands.sh

```diff
@@ -170,21 +163,12 @@ get_kubectl() {
 
 # Download and install microshift
 get_microshift() {
-    if [ $ARCH = "x86_64" ]; then
-        curl -L https://github.com/redhat-et/microshift/releases/download/$VERSION/microshift-linux-amd64 -o microshift
-        curl -L https://github.com/redhat-et/microshift/releases/download/$VERSION/release.sha256 -o release.sha256
-    fi
-
-    SHA=$(sha256sum microshift | awk '{print $1}')
-    if [[ $SHA != $(cat release.sha256 | awk '{print $1}') ]]; then echo "SHA256 checksum failed" && exit 1; fi
-
     sudo chmod +x microshift
     sudo mv microshift /usr/local/bin/
```

Comment:
I think I saw an error here on the last run, if so, could try
`sudo mv $HOME/microshift /usr/local/bin/`